Libraries/Text/TextInput/RCTBaseTextInputViewManager.m

```diff
@@ -47,6 +47,7 @@ @implementation RCTBaseTextInputViewManager {
 RCT_REMAP_NOT_OSX_VIEW_PROPERTY(returnKeyType, backedTextInputView.returnKeyType, UIReturnKeyType) // [macOS]
 RCT_REMAP_NOT_OSX_VIEW_PROPERTY(selectionColor, backedTextInputView.tintColor, UIColor) // [macOS]
 RCT_REMAP_OSX_VIEW_PROPERTY(selectionColor, backedTextInputView.selectionColor, UIColor) // [macOS]
+RCT_REMAP_OSX_VIEW_PROPERTY(cursorColor, backedTextInputView.cursorColor, UIColor) // [macOS]
```

Comment:
Might be better inside the ifdef block between lines 69 and 78, so that it's easier to not accidentally remove this line during a new upstream merge.